docs/configuration.md

```diff
@@ -33,7 +37,9 @@ lhci assert --preset=lighthouse:recommended --assertions.uses-webp-images=off
 
 ## Structure
 
-The structure of a `lighthouserc.json` is segmented by command. Any options you see for a particular command in the [CLI documentation](./cli.md) can be set by the property of the same name in the config file.
+The structure of the config file is segmented by command. Any options you see for a particular command in the [CLI documentation](./cli.md) can be set by the property of the same name in the config file.
+
+##### `lighthouserc.json`
```

Comment:
```suggestion
**`lighthouserc.json`:**
```

we've mostly been going with bold regular text for filenames within a section in the docs

Reply:
Done!